### ISSUE TITLE
Fermion tutorial 08 does not run fully

### ISSUE THREAD

**Issue by fretchen** [Qiskit-Extensions/qiskit-cold-atom#57]
### Informations

- **0.0.1**:
- **3.10**:
- **MacOS**:

### What is the current behavior?

The tutoiral `08_` cannot fully run through as one cell raises an error

### Steps to reproduce the problem

Attempt to run the following code in the notebook:

```
means_no_int = mean_counts_right(job_no_int, Ntimes)
means_int = mean_counts_right(job_int, Ntimes)
```

This raises the error:

```
QiskitColdAtomError: 'Expected length 4 for fermionic operator; received 0.'
```

### What is the expected behavior?

Not sure.

### Suggested solutions

Not sure.

**Comment by LaurinFischer** [comment]
Hi Fred, thanks for pointing this out. 

This looks to be some cross-compatibility issue between `qiskit-terra` and `qiskit-aer` raised from inside our simulator backend that rely on both terra and aer.  
The current versions of these are `qiskit-terra==0.23.1` and `qiskit-aer==0.11.2`.
I managed to reproduce your error with an older version of `terra` and the current one of `aer`.
However, using the most recent versions of both (or older ones of both) should be fine.

In summary, I got:

```{'qiskit-terra': '0.22.0', 'qiskit-aer': '0.11.2'}  ---> error ```
```{'qiskit-terra': '0.23.1', 'qiskit-aer': '0.11.2'} ---> no error```
```{'qiskit-terra': '0.22.0', 'qiskit-aer': '0.11.0' ---> no error```

Could you please confirm that this error is fixed by updating your terra and aer? Then we can update the requirements accordingly.
